src/components/Wizard/CreateVmWizard/providers/vmwareProviderPod.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Based on V2V Provider Pod manifest.yaml
+ */
+
+import { SecretModel, V2VVMwareModel } from '../../../../models';
+import { VCENTER_TYPE_LABEL, VCENTER_TEMPORARY_LABEL } from '../../../../constants';
+
+export const getDefaultSecretName = ({ username, url }) => {
+  if (!url) {
+    throw new Error('VMWare URL can not be empty.');
+  }
+
+  if (!url.startsWith('https://') && !url.startsWith('http://')) {
+    url = `https://${url}`;
+  }
+  const u = new URL(url || '');
```

Comment:
the url will never be falsy here, but nevertheless the `|| ''` would just throw an exception

Reply:
right, leftover, removing